<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PriorityQueueWindow</class>
 <widget class="QWidget" name="PriorityQueueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="removeTaskButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>540</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ùå  task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>‚Üê Go back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demoButton">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>üëÄ Demo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTunnelDeepButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tunnel (deep laying)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addRandomEventButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>540</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï Random game event</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskNameInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskListButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï  Task list</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>230</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>load:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loadStationDeepButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Station (deep laying)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>prior = </string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pageNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: orange;
	font-size: 35px;
	font-family: Roboto;
	background-color: transparent;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Priority queue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="taskListNameLabel">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>160</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #595959;
	font-size: 30px;
	font-family: Roboto;
	background-color: #d9d9d9;
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="loadStationShallowButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>280</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Station (shallow laying) </string>
   </property>
  </widget>
  <widget class="QLabel" name="pageNameLabel_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: grey;
	font-size: 35px;
	font-family: Roboto;
}</string>
   </property>
   <property name="text">
    <string> - Object work sequence</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï  task</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="taskListTable">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>220</y>
     <width>571</width>
     <height>660</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size: 25px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
	padding-left: 30px;
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskListNameInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loadTunnelShallowButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>400</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tunnel (shallow laying)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="taskPriorInput">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>458</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>600</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 17px;
	color: red;
	font-family: Roboto;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="demoModeLabel">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>10</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: orange;
	font-size: 35px;
	font-family: Roboto;
	background-color: transparent;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
